app/modules/conversations/conversation/conversation_service.py

```
@@ -218,18 +227,18 @@ async def create_conversation(
             return conversation_id, "Conversation created successfully."
         except IntegrityError as e:
             logger.error(f"IntegrityError in create_conversation: {e}", exc_info=True)
-            self.sql_db.rollback()
+            await self.sql_db.rollback()
             raise ConversationServiceError(
                 "Failed to create conversation due to a database integrity error."
             ) from e
```

Ruff (0.13.2)

Avoid specifying long messages outside the exception class

(TRY003)